web/src/styles/scss/layout.scss

```diff
@@ -70,3 +70,20 @@ footer {
     margin-top: 0;
   }
 }
+
+.alert--session-expiring {
+  position: sticky;
+  top: -200px;
```

Comment:
we will want to refactor our layout so we don't have to use these hacks

Reply:
completely agree